src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1011,6 +1011,10 @@ public static CriticalHitEvent getCriticalHit(Player player, Entity target, bool
         return null;
     }
 
+    /**
+     * @deprecated See {@link ForgeEventFactory#onAdvancementEarnedEvent} and {@link ForgeEventFactory#onAdvancementProgressedEvent}
+     */
+    //todo: remove and use ForgeEventFactory.onAdvancementEarned instead in 1.20
```

Comment:
Please replace the TODO with a deprecation annotation marking it for removal. As we already patch in the new event factory to the same place, this only needs to be removed.
```suggestion
    @Deprecated(forRemoval = true, since = "1.19.2")
```